hubblestack/splunklogging.py

```diff
@@ -41,25 +41,18 @@
 import json
 import time
 import copy
-import urllib3
-urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+from hubblestack.hec import http_event_collector, get_splunk_options
 
 import logging
 
-_max_content_bytes = 100000
-http_event_collector_debug = False
-
-hec = None
-
-
 class SplunkHandler(logging.Handler):
     '''
     Log handler for splunk
     '''
     def __init__(self):
         super(SplunkHandler, self).__init__()
 
-        self.opts_list = _get_options()
+        self.opts_list = get_splunk_options('hubblestack:returner:splunk')
```

Comment:
The argument here is unnecessary, right? Because it defaults to this value?

Reply:
yeah, that's the default. I think originally I planned to be really specific, but later learned the sourcetype options are name-prefixed in that same options namespace.